galera/Project1/Galera.cpp

```diff
@@ -32,3 +32,65 @@ int Galera::getTotalSallary() const
 {
     return _totalSallary;
 }
+
+bool Galera::hireEmployee(Employee first)
+{
+    bool k = true;
+    if (_employeeList.empty())
+    {
+        _employeeList.push_back(first);
+        return true;
+    }
+    auto x = _employeeList.begin();
+    for (int i = 0; i < _employeeList.size(); i++)
+    {
+        
+        if (first == *x)
+        {
+            return false; 
+        }
+        ++x;
+    }
+    if (k == true)
```

Comment:
You don't need to make this check. `k` would always be `true`, you could simply remove `k` at all.